src/bitmessagekivy/tests/test_filemanager.py

```diff
@@ -0,0 +1,70 @@
+from .telenium_process import TeleniumTestProcess
+from .common import ordered
+
+
+class FileManagerOpening(TeleniumTestProcess):
+    """File-manager Opening Functionality Testing"""
+    @ordered
+    def test_open_file_manager(self):
+        """Opening and Closing File-manager"""
+        # Checking current Screen(Inbox screen)
+        self.assert_wait_no_except('//ScreenManager[@current]', timeout=15, value='inbox')
+        # Method to open side navbar
+        self.open_side_navbar()
+        # Click to open Address Dropdown
+        self.assertExists('//NavigationItem[0][@text=\"dropdown_nav_item\"]', timeout=2)
+        self.assertExists(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"]', timeout=1
+        )
+        is_open = self.cli.getattr(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"][@is_open]', "is_open")
+        self.assertEqual(is_open, False)
```

Comment:
following JSON syntax, i.e. lowercase `false`, may fix it

Reply:
Added assertExist with json syntax `false`